examples/ui-demo/src/components/small-cards/SolanaNftCard.tsx

```diff
@@ -0,0 +1,226 @@
+            updateAuthority: publicKey,
+            mint: mint,
+            mintAuthority: publicKey,
+            name: tokenMetadata.name,
+            symbol: tokenMetadata.symbol,
+            uri: tokenMetadata.uri,
+          }),
+          ...metaData.map(([key, value]) =>
+            createUpdateFieldInstruction({
+              programId: TOKEN_2022_PROGRAM_ID,
+              metadata: mint,
+              updateAuthority: publicKey,
+              field: key,
+              value: value,
+            })
+          ),
+        ],
+      });
+
+      console.log(`Created transaction: ${tx.hash} 
+      https://explorer.solana.com/tx/${tx.hash}?cluster=devnet 
+      https://explorer.solana.com/address/${mint.toBase58()}?cluster=devnet
+    `);
+
+      setTransactionState("complete");
+    } catch (error) {
+      console.log(error);
+      setToast({
+        type: "error",
+        text: "Error sending transaction",
+        open: true,
+      });
+    }
+  };
+
+  const imageSlot = (
+    <div className="w-full h-full bg-[#DCFCE7] flex justify-center items-center relative">
+      <Image
+        src="/images/duckImage.png"
+        alt="Solana Duck NFT"
+        width={300}
+        height={300}
+        className="w-full h-full object-cover object-top"
+      />
+    </div>
+  );
+
+  const states = [
+    {
+      state: "signing",
+      text: "Signing transaction...",
+      isCompleteStates: ["complete", "sponsoring"],
+    },
+    {
+      state: "sponsoring",
+      text: "Sponsoring gas & minting NFT...",
+      isCompleteStates: ["complete"],
+    },
+  ];
+
+  const renderTransactionStates = (
+    <div className="flex flex-col gap-3">
+      {states.map(({ state, text, isCompleteStates }) => {
+        const isComplete = isCompleteStates.includes(transactionState);
+        return (
+          <div key={state} className="flex items-center gap-2">
+            {isComplete && (
+              <CheckCircleFilledIcon className="h-4 w-4 fill-demo-surface-success" />
+            )}
+            {!isComplete && <LoadingIcon className="h-4 w-4" />}
+            <p className="text-sm text-fg-secondary">{text}</p>
+          </div>
+        );
+      })}
+    </div>
+  );
+  const content = (
+    <>
+      {transactionState === "idle" || transactionState === "complete" ? (
+        <>
+          <p className="text-fg-primary text-sm mb-3">
+            Transact with one click using gas sponsorship and background
+            signing.
+          </p>
+          <div className="flex justify-between items-center">
+            <p className="text-fg-secondary text-sm">Gas Fee</p>
+            <p>
+              <span className="line-through mr-1 text-sm text-fg-primary">
+                $0.02
+              </span>
+              <span className="text-sm bg-gradient-to-r from-[#FF9C27] to-[#FD48CE] bg-clip-text text-transparent">
+                Free
+              </span>
+            </p>
+          </div>
```

Comment:
styling nit: can we make this consistent w/ the eth mint card? the text on the the eth card is larger (`text-base` instead of `text-sm`) and it behaves differently at different breakpoints (`hidden xl:flex`). i don't have any opinion about which is better, but i think we should be consistent.

Reply:
Cool thanks